rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -12,6 +12,7 @@
 #include <QScreen>
 #include <QUrl>
 
+#include "Emu/system_config_types.h"
```

Comment:
remove this line. unnecessary and wrong placement anyway

Reply:
sorry, my IDE did that